qendpoint-core/src/main/java/com/the_qa_company/qendpoint/core/triples/impl/BitmapTriples.java

```diff
@@ -472,6 +474,7 @@ public void load(InputStream input, ControlInfo ci, ProgressListener listener) t
 
 	@Override
 	public void mapFromFile(CountInputStream input, File f, ProgressListener listener) throws IOException {
+		log.info("Mapping BitmapTriples from {}", f.getName());
```

Comment:
I found this useful during the application startup to see that the application is not hanging but in fact loading a large file.